### DISCUSSION TITLE
How to bind custom domain?

### DISCUSSION THREAD

**Discussion by xmj02** [domcloud/domcloud#64] [General]
I have bought a domain: 51ziyuan.online
And I have upload my web site content to Dom Cloud.
Now I am following this doc https://domcloud.co/docs/features/dns trying to bind my domain to DOM server.

Step 1, In the registrar where you buy the domain, I have changed the name server to:
`
nsp.domcloud.co
nss.domcloud.co
`

Step 2,  I ran the deployment script

features:
- dns

As result, I saw these DNS Records set up.
![image](https://github.com/domcloud/domcloud/assets/150216391/c9e11e04-ee0e-4d9e-a79d-8477ddc5689a)

But now  when I am trying to access web site 51ziyuan.online,  just got error: cannot display the page or so...

I must have miss something?
Any comments are welcome.

**Comment by xmj02** [comment]
When I tried to access my web site, just got an error like this.
![image](https://github.com/domcloud/domcloud/assets/150216391/6300d998-0164-4308-9275-8559ac436800)

**Comment by willnode** [comment]
Hello, make sure you change the website's domain name via Portal. I just added this information to the documentation: https://domcloud.co/docs/features/dns

**Reply by ghost** [reply]
Got it. Thanks.
